text/3729-sized-hierarchy.md

```diff
@@ -1622,7 +1655,22 @@ longer relevant][zulip_issue_regions_too_simplistic].
 # Unresolved questions
 [unresolved-questions]: #unresolved-questions
 
-None currently.
+- Which syntax should be used for opting out of a default bound with const traits and
+  a trait hierarchy?
+  - This RFC is primarily written proposing the "positive bounds" approach, where
+    introducing a positive bound for a supertrait of the default bound will remove
+    the default bound.
+  - Alternatively, described in [*Adding `?ValueSized`*][adding-valuesized], existing
+    relaxed bounds syntax could be used, where a desired bound is written as opting out
+    of the next strictest.
+- Should `MetaSized` be introduced?
```

Comment:
```suggestion
- Should `MetaSized` be introduced instead of `ValueSized`?
```
That is the question, right?

Reply:
We *could* have both, I think replacing `ValueSized` makes sense but I don't think it matters too much to leave that open-ended here.

Reply:
Right now the RFC sounds like the open question is whether we have just `ValueSized` or both. So I think that should be clarified.

Reply:
Pushed a commit which should clarify this